pkg/kclient/operators.go

```diff
@@ -18,29 +19,37 @@ const (
 	apiVersion = "odo.dev/v1alpha1"
 )
 
-// GetClusterServiceVersionList returns a list of CSVs in the cluster
+// IsSBRSupported checks if resource of type service binding request present on the cluster
+func (c *Client) IsSBRSupported() (bool, error) {
+	return c.IsResourceSupported("apps.openshift.io", "v1alpha1", "servicebindingrequests")
+}
+
+// IsCSVSupported checks if resource of type service binding request present on the cluster
+func (c *Client) IsCSVSupported() (bool, error) {
+	return c.IsResourceSupported("operators.coreos.com", "v1alpha1", "clusterserviceversions")
+}
+
+// ListClusterServiceVersion returns a list of CSVs in the cluster
 // It is equivalent to doing `oc get csvs` using oc cli
-func (c *Client) GetClusterServiceVersionList() (*olm.ClusterServiceVersionList, error) {
+func (c *Client) ListClusterServiceVersion() (*olm.ClusterServiceVersionList, error) {
```

Comment:
This is a nitpick. Can we please name it `ListClusterServiceVersions`? Just beacuse we're trying to list all csvs in the cluster.

Reply:
Fixed